README.md

```diff
@@ -1,89 +1,98 @@
-[![npm][npm]][npm-badge]
-[![Build][build]][build-badge]
-[![Coverage][cover]][cover-badge]
-[![Chat][chat]][chat-badge]
+[![NPM][npm]][npm-url]
+[![Deps][deps]][deps-url]
+[![Tests][build]][build-url]
+[![Coverage][cover]][cover-url]
+[![Standard Code Style][style]][style-url]
+[![Chat][chat]][chat-url]
 
-# PostHTML <img align="right" width="220" height="200" title="PostHTML logo" src="http://posthtml.github.io/posthtml/logo.svg">
+# PostHTML <img align="right" width="200" height="220" title="PostHTML" src="http://posthtml.github.io/posthtml/logo.svg">
```

Comment:
? :) plugin boilerplate has ```{width: 200 height: 220}```, i changed it solely for that reason, but other way round i meant we could use ```{width: 220, height: 200)``` in posthtml-plugin-boilerplate if it should stay as s here.